custom/icds_reports/ucr/data_sources/person_cases_v2.json

```diff
@@ -2211,6 +2211,48 @@
       {
         "column_id": "count",
         "type": "count"
+      },
+      {
+        "column_id": "aadhar_date",
+        "type": "expression",
+        "is_nullable": true,
+        "is_primary_key": false,
+        "create_index": true,
+        "datatype": "date",
+        "expression": {
+          "type": "nested",
+          "argument_expression": {
+            "type": "reduce_items",
+            "aggregation_fn": "first_item",
+            "items_expression": {
+              "type": "filter_items",
+              "items_expression": {
+                "type": "icds_get_case_history_by_date",
```

Comment:
This could be really expensive, if there are a lot of these forms submitted. How many do you expect?

May be worht writing a icds_get_first_case_property or something like that so that it doesnt hit riak too mcuh